sdks/java/core/src/main/java/org/apache/beam/sdk/metrics/Lineage.java

```diff
@@ -72,18 +72,18 @@ private static String wrapSegment(String value) {
    *
    * <ul>
    *   <li>{@code system:segment1.segment2}
-   *   <li>{@code system:routine:segment1.segment2}
+   *   <li>{@code system:subtype:segment1.segment2}
```

Comment:
also fix routine->subtype which is the official name of this part of FQN. I was confused by some internal doc which marked "routine" here